contributors/design-proposals/downward_api_resources_limits_requests.md

```diff
@@ -1,4 +1,4 @@
-# Downward API for resource limits and requests
+# Downward API for resources limits and requests
```

Comment:
In these files we both find "resources limits" and "resource limits" , inconformity or deliberation?